package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-pisp-scheme-adapter",
-  "version": "11.1.1",
+  "version": "11.2.1",
```

Comment:
don't bump manually the version, there is `npm run release` ;-)

Reply:
done.